src/auth.py

```diff
@@ -331,3 +333,20 @@ def update_user_acls(
         acls_to_remove = current_user_acls - self.new_user_acls
         for acl in acls_to_remove:
             self.remove_acl(**asdict(acl))
+
+    def clear_users(self) -> None:
```

Comment:
**question** why is this needed? where is this used?

Reply:
@zmraul - Could you respond here?

Reply:
This method is meant to cover for leftover relations. It's meant to be used on update_status or similar, but I wasn't too sure on its usefulness, so it ended up not being used.

Reply:
Oh, I see. But if it is not used, we can probably remove it then, just to have less code to maintain and support